audit-service/src/unit-tests/java/uk/gov/gchq/palisade/service/audit/service/LoggerAuditServiceTest.java

```diff
@@ -172,15 +165,12 @@ public void auditReadRequestException() {
         verify(logger, atLeastOnce()).error(logCaptor.capture());
         final String log = logCaptor.getValue();
 
-        assertThat(log, allOf(
-                containsString(requestId.toString()),
-                containsString(resource.toString()),
-                containsString(resource.toString()),
-                containsString(exception.toString())
-        ));
-        assertThat(log, allOf(
-                containsString(LoggerAuditService.READ_REQUEST_EXCEPTION)
-        ));
+        assertAll(
```

Comment:
This is much easier to read and provides a better error message. Can also use `containsExactly` etc if only certain strings should be in the Iterable.
```
        assertThat(logCaptor.getAllValues()).containsAll(List.of(
                user.toString(),
                context.toString(),
                requestId.toString(),
                LoggerAuditService.REGISTER_REQUEST_COMPLETE));
```

Reply:
This method was tried to make it more readable but when running the tests locally they failed.

Reply:
I tried to implement this method of testing but when run locally the tests failed.

Reply:
That's strange. When I've had a problem with I noticed that I was testing ordering by mistake.

Reply:
The tests are now using
`        assertThat(logCaptor.getValue()).contains(
                user.toString(),
                context.toString(),
                requestId.toString(),
                LoggerAuditService.REGISTER_REQUEST_COMPLETE
        );`